internal/bundler/bundler.go

```diff
@@ -469,7 +477,49 @@ func bundlePython(ctx BundleContext, dir string, outdir string, theproject *proj
 		}
 		config["app"] = app
 	}
-	return os.WriteFile(filepath.Join(outdir, "config.json"), []byte(cstr.JSONStringify(config)), 0644)
+	if err := os.WriteFile(filepath.Join(outdir, "config.json"), []byte(cstr.JSONStringify(config)), 0644); err != nil {
+		return err
+	}
+	
+	if err := copyPromptsFromSrc(ctx.Logger, dir, outdir); err != nil {
+		return fmt.Errorf("copy prompts.yaml: %w", err)
+	}
+	
+	return nil
+}
+
+func copyPromptsFromSrc(logger logger.Logger, projectDir, outdir string) error {
+	srcRoot := filepath.Join(projectDir, "src")
```

Comment:
can't use source directly like this. must go through the ProjectContext to get the correct source folder otherwise this will break for python